README.md

```diff
@@ -34,6 +34,11 @@ External documentation:
 Please refer to the [release notes](#release-notes) for the supported CityEngine version.
 ### Installation
 
+#### Software Requirements
+- Windows 8.1, 10 and 11 (64bit)
```

Comment:
My bad, Windows 8.1 is not supported by Maya 2022 - let's remove it altogether.

Reply:
removed.